.changes/v3.3.0/673-improvements.md

```diff
@@ -0,0 +1,2 @@
+* Testing - add optional test config variable Misc.LdapContainer to override default LDAP  container location. 
+  (can be used to overcome registry throttling issues) [GH-673]
```

Comment:
```suggestion
  (can be used to overcome Terraform Registry throttling issues) [GH-673]
```

Reply:
This is "docker registry". I have updated this comment and as well more details in https://github.com/vmware/terraform-provider-vcd/pull/673#discussion_r640867736

Reply:
My bad, thank you for adding explanations!